packages/blocks/src/api/parser.js

```diff
@@ -438,6 +438,28 @@ export function createBlockWithFallback( blockNode ) {
 		name = Platform.OS === 'web' ? 'core/social-link' : name;
 	}
 
+	// Convert derivative blocks such as 'core-embed/instagram' to the
+	// canonical form 'core/embed'.
+	if ( name && name.indexOf( 'core-embed/' ) === 0 ) {
+		// Capture `core-embed/instagram` into `{"providerNameSlug":"instagram"}`
+		const providerSlug = name.substring( 11 );
+		const deprecated = {
+			speaker: 'speaker-deck',
+			polldaddy: 'crowdsignal',
+		};
+		attributes.providerNameSlug = Object.keys( deprecated ).includes(
+			providerSlug
+		)
```

Comment:
Instead of `Object.keys( a ).includes( b )`, might as well use `b in a` or `a.hasOwnProperty( b )`.